packages/core/src/api/createProcessor.ts

```diff
@@ -75,7 +76,7 @@ export function coreCreateProcessor(project: Project, options: RunGraphOptions)
     throw new Error(`Graph not found, and no main graph specified.`);
   }
 
-  const processor = new GraphProcessor(project, graphId as GraphId, options.registry);
+  const processor = new GraphProcessor(project, graphId as GraphId, options.registry, options.includeTrace);
```

Comment:
I think it would be preferred for optional properties to be grouped into an options object so they can be specified in any order, but this class is exported out of core which technically makes it part of the public interface. While I think most users would use it through createGraphProcessor we can't update this without a breaking change, but we can at least note this for the future